[MQTT] Cannot send STOP to generic roller shutter

<!-- Provide a general summary of the issue in the *Title* above -->
<!-- If the issue is related to a binding, please include its short name in -->
<!-- square brackets in the title - Example: "[astro] My issue..." -->

<!-- Important: Please contact the openHAB community forum for questions or -->
<!-- for configuration and usage guidance: https://community.openhab.org -->

<!-- Feel free to delete any comment lines in the template (starting with "<!--") -->

Since I upgraded from 4.0.4 to 4.1.1, I can't send STOP command to my mqtt channel configured as roller shutter. UP and DOWN still work

## Expected Behavior
<!-- If you're describing a bug, tell us what should happen -->
<!-- If you're suggesting a change/improvement, tell us how it should work -->
Send a message "STOP" to the topic linked to a generic roller shutter MQTT channel when the related item got the STOP command

## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior -->
<!-- Include related log information (preferably debug level) and related configs -->
<!-- Use a file attachment for log and config information longer than a few lines -->
<!-- Enclose multi-line log/code snippets with ``` on new lines for proper formatting -->
<!-- If suggesting a change/improvement, explain the difference from current behavior -->
<!-- For improvements, discuss at community.openhab.org first and include link to topic -->
sendCommand(STOP) to the item responds OK but no message in the mqtt topic.

In the logs, I got this exception :
```
[ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.handleCommand()' on 'org.openhab.binding.mqtt.generic.internal.handler.GenericMQTTThingHandler@8ec84e': STOP is not a valid command for MQTT.
java.lang.IllegalArgumentException: STOP is not a valid command for MQTT.
	at org.openhab.binding.mqtt.generic.values.RollershutterValue.parseType(RollershutterValue.java:97) ~[?:?]
	at org.openhab.binding.mqtt.generic.values.RollershutterValue.parseCommand(RollershutterValue.java:137) ~[?:?]
	at org.openhab.binding.mqtt.generic.ChannelState.publishValue(ChannelState.java:369) ~[?:?]
	at org.openhab.binding.mqtt.generic.AbstractMQTTThingHandler.handleCommand(AbstractMQTTThingHandler.java:153) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.openhab.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:147) [bundleFile:?]
	at org.openhab.core.internal.common.InvocationHandlerSync.invoke(InvocationHandlerSync.java:59) [bundleFile:?]
	at jdk.proxy479.$Proxy594.handleCommand(Unknown Source) [?:?]
	at org.openhab.core.thing.internal.profiles.ProfileCallbackImpl.handleCommand(ProfileCallbackImpl.java:95) [bundleFile:?]
	at org.openhab.core.thing.internal.profiles.SystemDefaultProfile.onCommandFromItem(SystemDefaultProfile.java:49) [bundleFile:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.openhab.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:147) [bundleFile:?]
	at org.openhab.core.internal.common.Invocation.call(Invocation.java:52) [bundleFile:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:840) [?:?]
```

## Possible Solution
<!-- Not obligatory, but suggest a fix/reason for the bug, -->
<!-- or ideas how to implement the addition or change -->
I managed to fix on my side by deleting `&& stopCommandString != null` in RollershutterValue.java:94 that comes from https://github.com/openhab/openhab-addons/pull/15875/
I guess it is only a workaround and the problem may be deeper

## Steps to Reproduce (for Bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps to -->
<!-- reproduce this bug. Include code to reproduce, if relevant -->
1. Create a Generic MQTT Thing
2. Add a channel type Rollershutter (default configuration is good)
3. Create an item of type Rollershutter linked to that channel
4. Send command STOP to that item

## Context
<!-- How has this issue affected you? What are you trying to accomplish? -->
<!-- Providing context helps us come up with a solution that is most useful in the real world -->
I saw the introduction of the bug was about mqtt.homeassistant. In my case, I only use mqtt.generic because I want to keep it simple and low level

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version used: 4.1.1
* Environment name and version : openjdk 17.0.9, node 16.20.2
* Operating System and version : Openhabian (Raspbian 11)

--- a/bundles/org.openhab.binding.mqtt.generic/src/main/resources/OH-INF/config/rollershutter-channel-config.xml
+++ b/bundles/org.openhab.binding.mqtt.generic/src/main/resources/OH-INF/config/rollershutter-channel-config.xml
@@ -94,7 +94,6 @@
 		<parameter name="stop" type="text">
 			<label>Stop Command</label>
 			<description>A string (like "STOP") that is sent when commanding the rollershutter to stop.</description>
-			<default>STOP</default>
 			<advanced>true</advanced>
 		</parameter>
 		<parameter name="onState" type="text">

